extensions/applicationinsights-analytics-js/src/JavaScriptSDK/AnalyticsPlugin.ts

```diff
@@ -265,6 +266,8 @@ export class AnalyticsPlugin extends BaseTelemetryPlugin implements IAppInsights
                     let inPv = pageView || {};
                     _pageViewManager.trackPageView(inPv, {...inPv.properties, ...inPv.measurements, ...customProperties});
         
+                    _resetAjaxAttempts();
```

Comment:
Move this to locations that are currently using the local variable defined in this file called _trackAjaxAttempts, and remove this local variable